go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/dell/gocsi v1.7.0
-	github.com/dell/gopowerstore v1.11.0
+	github.com/dell/gopowerstore v1.12.1-0.20230810192103-bc95a9a4b424
 	github.com/fsnotify/fsnotify v1.5.1
```

Comment:
Is there no update to other modules?

Reply:
I was surprised as well, and double checked my steps. There are some issues with the opentelemetry modules being flagged as missing (looks like https://github.com/open-telemetry/opentelemetry-go/issues/2938). I tried updating them but ran into some build errors. I don't want to update those otel modules this late so will investigate post 1.8.